contributing_to_docs/doc_guidelines.adoc

```diff
@@ -382,10 +382,49 @@ include::modules/module-filename.adoc[leveloffset=+1] <1>
 
 You can use `tag` attributes to link:https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions[define regions of a file and include by tag instead of the whole file.] This enables you to make conditional modifications to existing modules instead of duplicating content. When creating a tag, use a tag name that makes it easy to understand where the content defined will be included.
 
-.Tagged module include syntax
```

Comment:
This whole section about tags has gotten pretty big since its initial inclusion and also is pretty specific / probably not used all that much. Could it be moved to be a new subheading/section of the "Writing assembly" heading? Easier to find from TOC and keeps the content distinct.